### PR TITLE
module for ec2 poweruser role

### PR DESCRIPTION
**Pull request by edulop91**
useful for packer builds

### CODE REVIEW

File: aws-iam-role-ec2-poweruser/main.tf [hunk 1]
```diff
@@ -0,0 +1,42 @@
+resource "aws_iam_role" "ec2-poweruser" {
+  name = "${var.role_name}"
+
```
Comment (ryanking) [inline aws-iam-role-ec2-poweruser/main.tf]: we should take an iam_path

File: aws-iam-role-ec2-poweruser/main.tf [hunk 2]
```diff
@@ -0,0 +1,42 @@
...
+    "Action": "sts:AssumeRole"
+  }
+}
+EOF
+}
+
+resource "aws_iam_policy" "ec2" {
+  name        = "${var.role_name}-ec2"
+  description = "Provides full access to the ec2 api"
+
```
Comment (ryanking) [inline aws-iam-role-ec2-poweruser/main.tf]: iam_path

File: aws-iam-role-ec2-poweruser/outputs.tf [hunk 3]
```diff
@@ -0,0 +1,3 @@
+output "arn" {
```
Comment (ryanking) [inline aws-iam-role-ec2-poweruser/outputs.tf]: output `role_name` too

File: aws-iam-role-ec2-poweruser/main.tf [hunk 4]
```diff
@@ -0,0 +1,42 @@
...
+  policy = <<EOF
+{
+  "Version": "2012-10-17",
+  "Statement": [
+    {
+      "Sid": "ssm",
```
Comment (edulop91) [inline aws-iam-role-ec2-poweruser/main.tf]: renaming

**Pull request merged**